topologytest/hlv_test.go

```diff
@@ -95,16 +196,67 @@ func TestHLVUpdateDocumentSingleActor(t *testing.T) {
 			}
 			peers, _ := setupTests(t, tc.topology, tc.activePeerID)
 
+			docID := tc.docID("")
 			body1 := []byte(fmt.Sprintf(`{"peer": "%s", "topology": "%s", "write": 1}`, tc.activePeerID, tc.description()))
-			createVersion := peers[tc.activePeerID].CreateDocument(tc.collectionName(), tc.docID(), body1)
+			createVersion := peers[tc.activePeerID].CreateDocument(tc.collectionName(), docID, body1)
```

Comment:
should these documents return `BodyAndVersion` so you don't have to construct it everytime?